draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -234,22 +231,21 @@
      between generation of Link State PDUs. A source Intermediate 
      system shall wait at least this long before re-generating one
      of its own Link State PDUs.
-  The recommended value was 30 seconds.
 
   minimumLSPTransmissionInterval - This is the amount of time an 
      Intermediate system shall wait before further propagating 
      another Link State PDU from the same source system. 
-  The recommended value was 5 seconds.
 
   partialSNPInterval - This is the amount of time between periodic 
      action for transmission of Partial Sequence Number PDUs.
      It shall be less than minimumLSPTransmission-Interval.
-  The recommend value was 2 seconds.
-
 ]]></artwork>
         </figure></t>
 
-      <t>Most relevant to a discussion of LSP flooding rate is the recommended
+	<t>
+	  The recommended values were 30 seconds, 5 seconds, and 2 seconds, respectively.
```

Comment:
I preferred the original text where the recommended values were
associated with the individual parameter definition.

Reply:
No preference.